rsocket/transports/tcp/TcpDuplexConnection.cpp

```diff
@@ -210,7 +211,7 @@ class TcpInputSubscription : public Subscription {
 
   void request(int64_t n) noexcept override {
     DCHECK(tcpReaderWriter_);
-    DCHECK(n == kMaxRequestN)
+    DCHECK_EQ(n, std::numeric_limits<int64_t>::max())
```

Comment:
Frame_Request_N's constructor checks that the N value is to be smaller than INT_MAX. Would it be a problem to pass int64_max in here?